gene_finder.py

```diff
@@ -2,7 +2,7 @@
 """
 YOUR HEADER COMMENT HERE
```

Comment:
In future mini projects, please leave a header comment that describes what this code is about. It's going to help you a lot when the code base gets large.